Sources/Networking/Operations/Handling/CustomerInfoResponseHandler.swift

```diff
@@ -15,10 +15,46 @@ import Foundation
 
 class CustomerInfoResponseHandler {
 
-    init() { }
+    private let purchasedProductsFetcher: PurchasedProductsFetcherType
+    private let productEntitlementMapping: ProductEntitlementMapping?
+    private let customerInfoCreator: CustomerInfo.OfflineCreator
```

Comment:
I wonder if we could abstract all these 3 objects to a single `OfflineCustomerInfoCreator` or `OfflineCustomerInfoCalculator` as in Android... That way we abstract all the process needed to create the offline customer info from these handlers and operations

Reply:
Done, cleaned up this PR a lot.